internal/controller/olsconfig_controller.go

```diff
@@ -110,6 +110,10 @@ type OLSConfigReconcilerOptions struct {
 // NetworkPolicy for restricting access to OLS pods
 // +kubebuilder:rbac:groups=networking.k8s.io,namespace=openshift-lightspeed,resources=networkpolicies,verbs=get;list;watch;create;update;patch;delete
 
+// PVC access for the Postgres PVC
+// +kubebuilder:rbac:groups=core,namespace=openshift-lightspeed,resources=persistentvolumeclaims,verbs=get;list;watch;create;update;patch;delete
```

Comment:
@syedriko operator can be installed in any namespace , we will not have openshift-lightspeed namespace at that time.

Reply:
@xrajesh PTAL. I removed `namespace=openshift-lightspeed` everywhere in that file and replaced `groups=core` with `groups=""` as it's more idiomatic.